app/src/main/java/io/appium/uiautomator2/model/settings/ShouldUseCompactResponses.java

```diff
@@ -37,7 +36,10 @@ public Boolean getValue() {
 
     @Override
     protected void apply(Boolean shouldUseCompactResponses) {
-        Logger.debug("Dummy setting. Maintained in Session.capabilities.");
+        AppiumUIA2Driver
+                .getInstance()
+                .getSessionOrThrow()
+                .setCapability(SETTING_NAME, shouldUseCompactResponses);
     }
-
+    
```

Comment:
extra spaces?

Reply:
it's idea formatter, perhaps a bug there %/